image/image.go

```diff
@@ -122,6 +148,31 @@ type genericManifest interface {
 	GetLayers() []string
 }
 
+type descriptor struct {
+	MediaType string `json:"mediaType"`
+	Size      int64  `json:"size"`
+	Digest    string `json:"digest"`
+}
+
+type manifestSchema2 struct {
+	Config descriptor   `json:"config"`
+	Layers []descriptor `json:"layers"`
+}
+
+func (m *manifestSchema2) String() string {
+	// TODO: does it make sense to have string here?
+	return ""
+}
+
+func (m *manifestSchema2) GetLayers() []string {
+	blobs := []string{}
+	for _, layer := range m.Layers {
+		blobs = append(blobs, layer.Digest)
+	}
+	blobs = append(blobs, m.Config.Digest)
```

Comment:
I'd like to sort this out before moving to containers/image - so when a v2s2 manifest is retrieved and we look for layers I'm appending the config digest here because it's needed (as it was in v2s1, but v2s1 had the config embedded into the manifest itself) so now we _have_ _to_ retrieve the config somehow. Would it be better by adding a `Config()` to the `generalManifest` interface? or is this ok? /cc @mtrmac